opm/simulators/wells/GasLiftGroupInfo.cpp

```diff
@@ -346,10 +346,11 @@ std::tuple<double, double, double, double>
 GasLiftGroupInfo::
 initializeGroupRatesRecursive_(const Group &group)
 {
-    double oil_rate = 0.0;
-    double water_rate = 0.0;
-    double gas_rate = 0.0;
-    double alq = 0.0;
+    std::array<double,4> rates{};
+    double& oil_rate = rates[0];
+    double& water_rate = rates[1];
+    double& gas_rate = rates[2];
+    double& alq = rates[3];
```

Comment:
Use structured bindings?
```
double & [oil_rate, water_rate, gas_rate, alq] = rates;
```

Reply:
That is looking nice, almost hard to believe it is c++!

Reply:
i like it. structured bindings only works with auto though but i don't think that was your main point.